src/Sulu/Bundle/AdminBundle/Resources/js/containers/Login/types.js

```diff
@@ -1,3 +1,16 @@
 // @flow
 
 export type FormTypes = 'login' | 'reset-password' | 'forgot-password';
+
+export type ResetPasswordFormData = {|
+    password: string,
+|};
+
+export type ForgotPasswordFormData = {|
+    user: string,
+|};
+
+export type LoginFormData = {|
+    password: string,
+    username: string,
+|};
```

Comment:
we need to omit the `|` if we want to allow additional properties 😇 

```suggestion
export type ResetPasswordFormData = {
    password: string,
};

export type ForgotPasswordFormData = {
    user: string,
};

export type LoginFormData = {
    password: string,
    username: string,
};
```